transformer_engine/pytorch/csrc/extensions/quantizer.cpp

```diff
@@ -181,21 +181,23 @@ std::pair<TensorWrapper, py::object> MXFP8Quantizer::create_tensor(
     } else {
       data = at::empty(torch_shape, opts);
     }
-    rowwise_scale_inv = at::empty({numel / last_dim, last_dim / MXFP8_BLOCK_SIZE}, opts);
+    auto sinv0 = roundup(numel / last_dim, 128);
+    auto sinv1 = roundup(last_dim / MXFP8_BLOCK_SIZE, 4);
```

Comment:
It would be safer to check before assuming the dims are divisible by 32. This can be done with:
```c++
NVTE_CHECK(last_dim % MXFP8_BLOCK_SIZE == 0
           && (numel / last_dim) % MXFP8_BLOCK_SIZE == 0,
           "MXFP8 requires tensor dims that are divisble by ", MXFP8_BLOCK_SIZE,
           " (got shape=", torch_shape, ")");
```